src/v2/api/index.md

```diff
@@ -2488,6 +2488,7 @@ Used to denote a `<template>` element as a scoped slot.
   - `css` - boolean, Whether to apply CSS transition classes. Defaults to `true`. If set to `false`, will only trigger JavaScript hooks registered via component events.
   - `type` - string, Specify the type of transition events to wait for to determine transition end timing. Available values are `"transition"` and `"animation"`. By default, it will automatically detect the type that has a longer duration.
   - `mode` - string, Controls the timing sequence of leaving/entering transitions. Available modes are `"out-in"` and `"in-out"`; defaults to simultaneous.
+  - `duration` - number | { enter: number, leave: number } Specify the duration of transition. By default, Vue waits for the first `transitionend` or `animationend` event on the root transition element
```

Comment:
Sorry I was late. There are several problems with this PR still:

* `enter` and `leave` should be in backticks
* A trailing period is missing for the last sentence
* "Specifies" should be "Specify" (it's incorrect currently for `type` as well)
* A comma is missing after }.

Reply:
Ok, should I open new PR?